README.md

```diff
@@ -12,6 +12,20 @@ This repository is currently Work In Progress but will eventually be used to man
 
 The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated).
 
+**NOTE**
+Acceptance of enhancement doesn't guarantee that the feature will land in the current or later release. The process is collaborative effort between contributor/s and Approvers. Features not landing to release branch prior to FF will be reverted based on their state (GA/Beta/Alpha, Off/On by default) that reflects how harmfull keeping the code is to the system.
+
+## Approvers responsibility
+
+1. After Feature Freeze, approvers needs to go over issue and perform the following checklist
+
+   1. All PRs are merged into release branch
+   2. Docs PR is merged (plan review ahead of release if only placeholder is opened)
+   3. Verify that Enhancement was implemented and doesn't need any update
+   4. Track any bugs
+2. Weekly check-in on progress of Enhancement and its implementation
+3. Coordinate sigs, reviewers and approvers in order to progress the Enhancement
```

Comment:
My opinion is:
**The VEP owner responsibility**: coordinate between SIGs.
**SIG responsibility**: Prioritize VEP, decide if to track or not for the upcoming release, assign reviewers/approvers.

In other words, while each SIG has its own responsibilities, the VEP owner should be the one to do the actual coordination. As an example, if a VEP is owned by sig-compute but demands an ACK from sig-storage, the VEP owner should be joining the sig-storage meeting and ask for help.